src/game/GridBox.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import { StyleSheet } from 'react-native';
+import Box from './Box';
+
+export default class GridBox extends Box {
```

Comment:
We usually avoid extending another Component like this. Instead, we use the component (`Box` here`) and pass the props we want.

```jsx
render() {
    return (
       <Box>
       </Box>
    );
}
```